script/Ciel/Day 15/15_01B_CIEL15B_1.txt

```diff
@@ -2792,13 +2797,13 @@ Not only that, but the sun feels strangely bright.
 -- Page 166, Offset 42467.
 -- 復元できているのは最低限の個所だけですから！　あと１時間はそのままでいてもらわないと！」
 // 'bare minimum amount of restoration' pretty clunky
-Your wounds have closed, but that's about it!
+Your wounds have only closed in a few vital spots, but that's about it!
  You need to stay like this for at least another hour!"
 }
 [sha:aaffd11899079474df65b106a563cd538a570f8f]{
 -- Page 167, Offset 42468.
 -- 「……先輩。矛盾したこと言ってるって、わかってます？」
-"Senpai... You realize you just contradicted yourself, right?"
+"Senpai... You realize you just contradicted what you said before, right?"
```

Comment:
Kinda prefer shorter ver

Reply:
it was too unclear before

Reply:
at least naturalize the dialog to 
> what you said a second ago

or something, I guess

Reply:
Uh, sure, but it seems like incredible hair-splitting